CHANGELOG.md

```diff
@@ -5,6 +5,9 @@
 * @iov/bcp-type: `BcpConnection.getNonce` and `.watchNonce` now return
   `Promise<Nonce>`/`Stream<Nonce>` and implementations set a default value on
   their own.
+* @iov/jsonrpc: Add new package for type-safe JSON-RPC 2.0 interfaces and
+  response parsers. This is used for out-of-process signing and can be reused
+  in @iov/ethereum and @iov/tendermint-rpc later on.
```

Comment:
Yes, I saw tendermint rpc is very similar to ethereum (probably based the design on it).
Would be great to reuse this package

Reply:
Fortunatly JSON-RPC 2.0 is a standard that both use. And as far as I can see, both Tendermint and Ethereum are spec-complient. Locally I could already use @iov/jsonrpc for Ethereum